express-rest-api/server/index.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
```

Comment:
You need to add a excute permission on this file by `chmod +x server/index.js`.
After that, you can change `"start": "node ./server/index.js"` to `"start": "./server/index.js"` in `package.json`.

Reply:
Do you think is it better to leave this line1? Do you mean change the` "start": "node ./server/index.js"` to `"start": "./server/index.js"` in `package.json`? BTW Thank you for all the support! ✨